Aliases/iwyu

```diff
@@ -0,0 +1 @@
+../Formula/include-what-you-use.rb
```

Comment:
Why?

Reply:
nm, I see.

Can you please confirm that this is the package name on debian/ubuntu?

Reply:
Yes, https://packages.debian.org/stretch/iwyu, https://packages.ubuntu.com/bionic/iwyu, etc.